Dockerfile

```diff
@@ -2,32 +2,26 @@ FROM node:14.16.1 AS make-bundle
 ENV WEB_APP=/home/app/webapp
 WORKDIR $WEB_APP
 COPY . $WEB_APP
-RUN npm install 
-RUN npm run webpack 
+RUN npm install && \
+    npm run webpack 
+
+
 
-# base image  
 FROM python:3.7  
-# setup environment variable  
 ENV WEB_APP=/home/app/webapp  
-
-# set work directory  
 RUN mkdir -p $WEB_APP  
-
-# where your code lives  
 WORKDIR $WEB_APP  
-
-# set environment variables  
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1  
-# install dependencies  
+ENV PYTHONDONTWRITEBYTECODE 1 \
+    PYTHONUNBUFFERED 1  
 RUN pip install --upgrade pip  
-# copy whole project to your docker home directory. 
-# COPY . $DockerHOME
-COPY --from=make-bundle /home/app/webapp $WEB_APP 
-# run this command to install all dependencies  
-RUN pip install -r requirements.txt  
-# port where the Django app runs  
+COPY --from=make-bundle /home/app/webapp/ $WEB_APP 
```

Comment:
my idea: split this one to 2:
1) copy only bundle.js
2) copy . $WEB_APP

and then you wont have the node_modules mess